### ISSUE TITLE
Добавить шаблоны полезных артефактов

### ISSUE THREAD

**Issue by etolstoy** [tlbootcamp/tlroadmap#137]
Столкнулся в своей деятельности с проблемой – хочется иметь готовый простой и удобный шаблон для составления плана развития тимлида с использованием нашей карты. Помимо плана развития, кажется, стоит сделать и еще несколько шаблонов:
- Должностные обязанности
- Требования к вакансии

В светлом будущем я вообще хотел бы завести какой-нибудь интерактивный конструктор для всей этой истории. Берешь, натыкиваешь себе нужных веточек, тебе на выходе заполненные документы.

Есть мысли, шаблоны каких артефактов могут еще пригодиться?

**Comment by DevAlloy** [comment]
Не очень понимаю как план развития бьется с должностными обязанностями и требованиям к вакансии. Это ведь разные вещи.

**Comment by etolstoy** [comment]
Разные, я про шаблоны для типовых задач.

**Comment by antonkhvorov** [comment]
@etolstoy например шаблон для meeting minutes и т.д.?
Возможно еще что-то более серьезное, тип архитектурных документов

**Comment by etolstoy** [comment]
Так, я не правильно объяснил все. Смотрите еще раз. Роадмап тимлида полезен не сам по себе, а как источник данных для:
- Формирования плана развития тимлида
- Описания вакансии тимлида
- Описания должностных инструкция для тимлида

Когда мне нужно было описать план развития, я столкнулся с тем, что удобного шаблона нет. То же и с должностной инструкцией - шаблон я каждый раз переизобретаю заново. Вот именно их я и хочу сделать.
